agent-core/src/main/java/com/newrelic/agent/android/crash/CrashReporter.java

```diff
@@ -106,7 +106,16 @@ protected void reportSavedCrashes() {
                     log.info("CrashReporter: Crash [" + crash.getUuid().toString() + "] has become stale, and has been removed");
                     StatsEngine.get().inc(MetricNames.SUPPORTABILITY_CRASH_REMOVED_STALE);
                 } else {
-                    reportCrash(crash);
+                    if (FeatureFlag.featureEnabled(FeatureFlag.OfflineStorage)) {
```

Comment:
Likewise for crash, the CrashReporter manages payloads of crash data. This is redundant.